config/initializers/active_merchant.rb

```diff
@@ -9,3 +9,21 @@
   ActiveMerchant::Billing::Gateway.wiredump_device = Rails.root.join('log/activemerchant.log').open('a')
   ActiveMerchant::Billing::Gateway.wiredump_device.sync = true
 end
+
+ActiveMerchant::Billing::StripeGateway.prepend(Module.new do
+  def headers(options = {})
+    key = options[:key] || @api_key
+    idempotency_key = options[:idempotency_key]
+
+    headers = {
+      'Authorization' => 'Basic ' + Base64.encode64(key.to_s + ':').strip.delete("\n"),
```

Comment:
All this is exactly the same as [the original code of this version of the gem](https://github.com/activemerchant/active_merchant/blob/b2f5e89eb383429d47e446f248d7bfe4f95ac3d0/lib/active_merchant/billing/gateways/stripe.rb#L611-L625) except that I am also deleting all `"\n"`.

Reply:
Failure: without the monkey patch
Success: with the monkey patch

![image](https://user-images.githubusercontent.com/11318903/102797663-4bcd8f80-43b0-11eb-8569-34edd78aaafd.png)

Reply:
Hmm so that failure is saying the API key provided was invalid. Have we fixed two problems here?

Reply:
@kevprice83 What do you mean? that this bug was also already in `master`?

Reply:
you did fix two bugs, @Martouta. https://issues.redhat.com/browse/THREESCALE-6533 🥇